bee-ledger/src/types/unspent.rs

```diff
@@ -37,7 +37,7 @@ impl Unspent {
 
 impl core::fmt::Display for Unspent {
     fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
-        write!(f, "{}", *self)
+        write!(f, "{}", self.0)
```

Comment:
```suggestion
        self.0.fmt(f)
```
I prefer doing this if I just want to "forward" the formatting. It's a bit less noisy and doesn't require macro expansion.

Reply:
I would rather not do this, or at least not now as the whole codebase uses the macro and it would be inconsistent. I personally prefer the macro as it's easier to maintain if we need to change the formatting.